migrations/versions/0310_account_change_type.py

```diff
@@ -0,0 +1,50 @@
+"""
+
+Revision ID: 0310_account_change_type
+Revises: 0305e_account_change_type
+Create Date: 2020-09-16 15:47:21.019759
+
+"""
+from flask import current_app
+from alembic import op
+
+revision = '0310_account_change_type'
+down_revision = '0309_system_templates_priority'
+
+template = {
+    'id': current_app.config['ACCOUNT_CHANGE_TEMPLATE_ID'],
+    'subject': 'Account information changed | Informations de compte modifiées',
+    'content_lines': [
+        'Your GC Notify user account information was changed on ((base_url)).',
+        '',
+        'Updated information: ((change_type_en))',
+        '',
+        'If you did not make this change, [contact us](((contact_us_url)) "contact us") immediately.',
+        '',
+        '___',
+        '',
+        "Les informations de votre compte d''utilisateur ont été modifiées : ((base_url)).",
+        '',
+        "Informations mises à jour : ((change_type_fr))",
+        '',
+        "Si vous n''avez pas effectué ce changement, [communiquez avec nous](((contact_us_url)) \"communiquez avec nous\") immédiatement.",
+    ]
```

Comment:
Used the template on staging, it was different in the local database.

Notable change is `change_type_en` + `change_type_fr`

Reply:
You used the staging template here? Can you confirm if this is the same as prod or not? If not, we'll want the prod one here.

Reply:
Confirmed it's the same in staging and prod 👌